ADApp/Db/NDCodec.template

```diff
@@ -217,3 +221,19 @@ record(waveform, "$(P)$(R)CodecError")
     field(FTVL, "CHAR")
     field(NELM, "256")
 }
+
+# Sets group of picture size. A value of 1 is effectively frame-level compression. Set to higher values for true video compression.
+record(ao, "$(P)$(R)GOPSize")
+{
+    field(DTYP, "asynInt32")
+    field(OUT,  "@asyn($(PORT),$(ADDR=0),$(TIMEOUT=1))GOP_SIZE")
+}
+
+# Sets the quantizer (qmin and qmax) to value written to PV.
+# One can effectively adjust quality this way, with 1 being the least lossy, and higher numbers lossier
+record(ao, "$(P)$(R)QMinMax")
+{
+    field(DTYP, "asynInt32")
+    field(OUT,  "@asyn($(PORT),$(ADDR=0),$(TIMEOUT=1))QMINMAX")
+}
```

Comment:
It is my understanding that QMin and QMax set a range for acceptable "Q" (which stands for quantizer and roughly corresponds to quality), and then the codec selects a "Q" within this range based on various other factors. When I was testing this I found it useful to have a single PV that changes the compression quality, rather than separate min/max that must be set every time. It would be more complete to have separate QMin/QMax PVs, and then perhaps a PV that controls a "QCenter" which shifts QMin and QMax by the same amount as a convenience, although there is a little bit of complexity in implementing "QCenter", and I tried to keep things simple for the initial implementation. I agree that in principle QMin and QMax should be exposed.

Reply:
Fair enough, that's a more complicated interface.